talent_scout.py

```
@@ -40,11 +39,7 @@ def audit_github(handle: str) -> Dict[str, str]:
         if user_resp.status_code == 401:
             raise ConfigurationError("GitHub API key is invalid or has expired.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)